src/web/app/pages-instructor/instructor-home-page/instructor-home-page.component.ts

```diff
@@ -386,4 +395,22 @@ export class InstructorHomePageComponent extends InstructorSessionModalPageCompo
   downloadSessionResultEventHandler(tabIndex: number, rowIndex: number): void {
     this.downloadSessionResult(this.courseTabModels[tabIndex].sessionsTableRowModels[rowIndex]);
   }
+
+  /**
+   * Retries loading all the courses
+   */
+  retryLoadingCourses(): void {
```

Comment:
I don't think we should create any new retry method. `this.loadCourses();` would already has the responsibility of setting `this.hasCoursesLoaded` and `this.hasCoursesLoadingFailed` to `false`.